### ISSUE TITLE
Enhancement: Add a permission for which players get their skin on join

### ISSUE THREAD

**Issue by Voiddar** [SkinsRestorer/SkinsRestorerX#236]
I was playing with the config and I found something quite interesting. The vanish incompatiility bug is fixed by just disabling the onJoin skins option (set "DisableOnJoinSkins" to true)
Now I wonder if a permission could be added so only the users who have the permission count for this config option. This would mean, only users who have the permission get their skin set automatically when they join, the rest doesn't

TLDR:
- Staffs who have access to vanish don't get onJoin skins
- default users do

**Comment by xknat** [comment]
it would not apply the skin on join. doubt you want that

**Comment by Voiddar** [comment]
The permission would only be set false for staff, so their vanish on join doesn't bug out.

**Comment by xknat** [comment]
@Voiddar Resolved on 763eea66fd04e473798ed1bc32c36dba65a4aa67
